docs/content/usage.md

```diff
@@ -27,46 +87,34 @@ All workflows are using the tagging convention listed below. It is subsequently
 Issue the following commands to acquire the necessary files:
 
 ``` BASH
-DMS_GITHUB_URL='https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master'
+TAG='v11.3.1'
+DMS_GITHUB_URL="https://github.com/docker-mailserver/docker-mailserver/blob/${TAG}"
 wget "${DMS_GITHUB_URL}/docker-compose.yml"
 wget "${DMS_GITHUB_URL}/mailserver.env"
-
-# Optional
-wget "${DMS_GITHUB_URL}/setup.sh"
-chmod a+x ./setup.sh
 ```
 
-## Create a docker-compose Environment
+Then edit `docker-compose.yml` to your liking; substitute `mail.example.com` according to your FQDN, and if you want to use SELinux for the `./docker-data/dms/config/:/tmp/docker-mailserver/` mount, append `-z` or `-Z`. Then configure the environment specific to the mail server by editing [`mailserver.env`][docs-environment], but keep in mind that:
```

Comment:
I collapsed this because point 1 was never suited to be there anyway and I felt it was more concise to just write it as text and not with an enumeration.

I can bring back the enumeration though, if you like it more, I will not be insisting on using the sentences.

Reply:
See f3b7fdc744a864a79f8ba9215b2ef101f0618282.